src/css/modules/_ApplicationBlock.scss

```diff
@@ -0,0 +1,98 @@
+.coa-ApplicationBlock__title {
+  text-transform: uppercase;
+  text-align: center;
+}
+
+#rCw.coa-ApplicationBlock__Recollect {
```

Comment:
I think it would be cool to isolate this Recollect override code in its own file with maybe an explanatory comment at the top.

Reply:
👍